tox.ini

```diff
@@ -29,6 +29,15 @@ deps =
     pytest-mock
     pytest-html
 
+[testenv:py27]
+deps =
+    pytest
+    pytest-cov
+    pytest-remove-stale-bytecode
+    pytest-mock
+    pytest-html
+    DateTime
```

Comment:
As `DateTime` is already ported to Python 3 it should be tested with at least one Python 3 version, too.